src/main/java/com/metamx/collections/bitmap/RoaringBitmapFactory.java

```diff
@@ -58,9 +61,7 @@ public MutableBitmap makeEmptyMutableBitmap()
   @Override
   public ImmutableBitmap makeEmptyImmutableBitmap()
   {
-    return new WrappedImmutableRoaringBitmap(
-        EMPTY_IMMUTABLE_BITMAP
-    );
+    return WRAPPED_IMMUTABLE_ROARING_BITMAP;
```

Comment:
@lemire The iterator shares the high-low container `private MappeableContainerPointer cp = ImmutableRoaringBitmap.this.highLowContainer
                .getContainerPointer();` and advances it https://github.com/lemire/RoaringBitmap/blob/master/src/main/java/org/roaringbitmap/buffer/ImmutableRoaringBitmap.java#L739 

I haven't finished digging through it completely yet (for example, how safe is `getContainerPointer` as far as creating a unique container pointer?) so it might still be ok.